src/ui/TopBanner/TopBanner.tsx

```diff
@@ -26,7 +26,7 @@ type Variants = keyof typeof variants
 
 const topBannerContext = z.object({
   variant: z.union([z.literal('default'), z.literal('warning')]),
-  localStorageKey: z.string(),
+  localStorageKey: z.string().nullish(),
```

Comment:
If it can only be set to a `string` or `undefined`, can this be swapped from `nullish` to `optional`?